src/DesignKit.tsx

```diff
@@ -2987,6 +2988,234 @@ const MessageSection: React.FC = () => {
+        <Columns>
+          <code>{'%elevation-base-dark'}</code>
+          <code>{'%elevation-base-light'}</code>
+          <code>{'%elevation-strong-dark'}</code>
+          <code>{'%elevation-strong-light'}</code>
+          <code>{'%elevation-contrast-light'}</code>
+          <code>{'%elevation-contrast-dark'}</code>
+          <code>{'%elevation-border-light'}</code>
+        </Columns>
+        <Columns>
+          <code>{'%elevation-border-dark'}</code>
+          <code>{'%elevation-hover-light'}</code>
+          <code>{'%elevation-hover-dark'}</code>
+          <code>{'%elevation-selected-light'}</code>
+          <code>{'%elevation-selected-dark'}</code>
+          <code>{'%elevation-backdrop'}</code>
+        </Columns>
+        <Columns>
+          <code>{'%text-title-light'}</code>
+          <code>{'%text-title-dark'}</code>
+          <code>{'%text-secondary-light'}</code>
+          <code>{'%text-secondary-dark'}</code>
+          <code>{'%text-disabled-light'}</code>
+          <code>{'%text-disabled-dark'}</code>
+          <code>{'%text-link-light'}</code>
+        </Columns>
+        <Columns>
+          <code>{'%elevation-shadow-small'}</code>
+          <code>{'%elevation-shadow-medium'}</code>
+          <code>{'%elevation-shadow-large'}</code>
+        </Columns>
+        <br />
+        <Paragraph>
+          base elevation without hover interaction (for elements elements at level 1)
+        </Paragraph>
+        <Paragraph>
+          composing with
+          <code>{'@extend %elevationBase<Light-or-Dark>;'}</code>
+          <code>{'@extend %elevationBorder<Light-or-Dark>;'}</code>
+          <code>{'@extend %elevationHover<Light-or-Dark>;'}</code>
+        </Paragraph>
+        <div className={css[`elevation1xNoHover${cssSulfix}`]} />
+        <br />
+        <Paragraph>
+          base elevation without hover interaction (for elements elements at level 1)
+        </Paragraph>
+        <Paragraph>the hover state has the sabe bg color as the selected state2</Paragraph>
+        <Paragraph>
+          composing with
+          <code>{'@extend %elevationBase<Light-or-Dark>;'}</code>
+          <code>{'@extend %elevationBorder<Light-or-Dark>;'}</code>
+          <code>{'@extend %elevationHover<Light-or-Dark>;'}</code>
+        </Paragraph>
+        <div className={css[`elevation1xNoHover${cssSulfix}`]} />
+        <br />
+        <Paragraph>
+          special elevation element with more contrast (to be used by special elements elements at
+          level 1)
+        </Paragraph>
+        <Paragraph>
+          composing with
+          <code>{'@extend %elevationBase<Light-or-Dark>;'}</code>
+          <code>{'@extend %elevationStrong<Light-or-Dark>;'}</code>
+          <code>{'@extend %elevationBorder<Light-or-Dark>;'}</code>
+        </Paragraph>
+        <div className={css[`elevationStrong${cssSulfix}`]} />
+        <br />
+        <Paragraph>tooltip-like element (with special contrast)</Paragraph>
```

Comment:
nit: "tooltip-like" doesn't sound good in this context, especially since there isn't anything here specific to tooltips